sdk/identity/identity/integration/AzureFunctions/RunTest/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "azure-function-test",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "build": "tsc",
```

Comment:
I think that's a great idea, as the Azure Functions (in none of their various ways to deploy) have a automatic build that happens. But for this one I wanted to mimic the exact thing that a customer would do. I think I'll probably try that for AKS and VM since we haven't finished that yet.